### PR TITLE
Add `PV Power` for ZCS Azzurro HYD ZSS HP

### PR DESCRIPTION
**Pull request by Edsol**

### CODE REVIEW

File: custom_components/solarman/inverter_definitions/zcs_azzurro-hyd-zss-hp.yaml [hunk 1]
```diff
@@ -14,6 +14,22 @@ parameters:
...
+        class: "power"
+        state_class: "measurement"
+        uom: "W"
+        rule: 1
+        digits: 0
+        registers: [0x0586, 0x0589]
+        sensors:
+          - scale: 10
+            registers: [0x0586]
+          - scale: 10
```
Comment (davidrapan) [inline custom_components/solarman/inverter_definitions/zcs_azzurro-hyd-zss-hp.yaml:29]: Hi @Edsol, you can "move" `scale: 10` up, outside of sensors list. 😉

**Comment by davidrapan** [comment]
Ah, I just noticed those:
* refactor: Update license
* refactor: Update funding.yml
* refactor: Update readme.md
* entity sensor translations
* refactor: Update funding.yml

which you are dragging with you PR again. I'm gonna need you to "refork" (or resync the repo in any other way you know of). And repeat PR or fix your branch, cause there are confllcts.

**Comment by Edsol** [comment]
I just noticed them too, I simply did a refork. I will fix it, thank you

File: custom_components/solarman/inverter_definitions/zcs_azzurro-hyd-zss-hp.yaml (hunk 1 above)
Comment (Edsol) [reply custom_components/solarman/inverter_definitions/zcs_azzurro-hyd-zss-hp.yaml]: fixed 👍

**Comment by Edsol** [comment]
> Ah, I just noticed those:
> 
> * refactor: Update license
> * refactor: Update funding.yml
> * refactor: Update readme.md
> * entity sensor translations
> * refactor: Update funding.yml
> 
> which you are dragging with you PR again. I'm gonna need you to "refork" (or resync the repo in any other way you know of). And repeat PR or fix your branch, cause there are confllcts.

Only the configuration yaml file has changed, is PR correct now?

**Pull request merged**